sdk/communication/azure-communication-phonenumbers/src/main/java/com/azure/communication/phonenumbers/PhoneNumbersAsyncClient.java

```diff
@@ -144,7 +231,7 @@
      * @return A {@link PollerFlux} object.
      */
     @ServiceMethod(returns = ReturnType.LONG_RUNNING_OPERATION)
-    public PollerFlux<PollResult<Void>, Void> beginPurchasePhoneNumbers(String searchId) {
+    public PollerFlux<Void, Void> beginPurchasePhoneNumbers(String searchId) {
```

Comment:
This should return PollerFlux<T, Void> where T is the result of the polling operation.

Reply:
Yup I will fix, just haven't gotten to this yet :)